src/taskManager/src/AbstractTask.cpp

```diff
@@ -20,6 +20,8 @@ template <unsigned int stackSize>
 void AbstractTask<stackSize>::start() {
     m_taskHandle = xTaskCreateStatic(wrapper, m_taskName, stackSize, this, m_priority,
                                      m_stackArray.data(), &m_taskBuffer);
+    m_taskHandle = xTaskCreateStatic(wrapper, m_taskName, stackSize, this, m_priority,
```

Comment:
Again, does the propolis lib works here?

Reply:
Changed to propolis